### PR TITLE
[2.1.0] Introduce the DescribeView ACPS API

### PR DESCRIPTION
**Pull request by ctwomblyamzn**
*Issue #, if available:*
N/A
*Description of changes:*
See commit message. 

Tested on the customer chat widget.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/client/client.spec.js [hunk 1]
```diff
@@ -51,4 +51,77 @@ describe("client test cases", () => {
       }
     });
   });
+
+  describe("Client Method Tests", () => {
```
Comment (spencerlepine) [inline src/client/client.spec.js:55]: internal code doesn't have this describe block, could we mirror it?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "amazon-connect-chatjs",
-  "version": "2.0.2",
+  "version": "2.1.0",
```
Comment (spencerlepine) [inline package.json:3]: can we update `package-lock.json` as well

**Review by spencerlepine — CHANGES_REQUESTED** [review]

File: src/client/client.spec.js (hunk 1 above)
Comment (ctwomblyamzn) [reply src/client/client.spec.js:55]: I'll publish a change to the internal code -- this is correct

File: package.json (hunk 2 above)
Comment (ctwomblyamzn) [reply package.json:3]: Yep!

**Review by spencerlepine — APPROVED** [review]

**Review by doreechi — APPROVED** [review]

**Pull request merged**